audio/cmus/Portfile

```diff
@@ -3,19 +3,21 @@
 PortSystem 1.0
 PortGroup           github 1.0
 
-github.setup        cmus cmus 2.7.1 v
-revision            4
+github.setup        cmus cmus 2.8.0 v
+revision            0
 categories          audio
-maintainers         {g5pw @g5pw} openmaintainer
+maintainers         {g5pw @g5pw} \
+                    {Traace @Traace} \
```

Comment:
I didn't mean *exactly* like g5pw. "Traace" is invalid here because you are not a Macports committer. Replace that with your original obfuscated email address.

Reply:
Man you said: "Add your GitHub handle here in brackets (like it was for g5pw)."

Please....